_data/projects/multiplayer-game-server.yml

```
@@ -0,0 +1,13 @@
+name: Multiplayer Game Server
+desc: A web app which supports multiplayer gaming like Snake n' Ladders and Connect four built using Python Flask backend
+site: https://github.com/YashasG98/Multiplayer-Game-Server
+tags:
+  - python
+  - html
+  - css
+  - flask
+  - mysql
+  - javascript
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/YashasG98/Multiplayer-Game-Server/issues
```

I think you can point this to the `up-for-grabs` label on the project:

```suggestion
  link: https://github.com/YashasG98/Multiplayer-Game-Server/labels/up-for-grabs
```